cli/runtime/apis/config/v1alpha1/clientconfig_types.go

```diff
@@ -34,116 +34,116 @@ const (
 )
 
 // Server connection.
-// Deprecation targeted for a a future version. Superseded by Context.
+// Deprecation targeted for a future version. Superseded by Context.
 type Server struct {
 	// Name of the server.
-	Name string `json:"name,omitempty" yaml:"name"`
+	Name string `json:"name,omitempty" yaml:"name,omitempty" mapstructure:"name,omitempty"`
```

Comment:
Out of curiosity, why are we adding all these `mapstructure` tags?
I don't see us importing `github.com/mitchellh/mapstructure` anywhere.
I'm not saying it is bad, I'm just trying to understand 😄

Reply:
removed. no longer used